src/tgbot_expenses/services/account_service.py

```diff
@@ -0,0 +1,124 @@
+from decimal import Decimal
+
+from sqlalchemy.sql import select
+
+from src.tgbot_expenses.database.db import AsyncSessionWithEnter, database
+from src.tgbot_expenses.models.expense_tracking_models import Account
+
+
+async def insert_account(account_name: str, account_amount: Decimal) -> None:
+    """
+    Insert a new entry into the 'accounts' table with the given account
+    name and account amount.
+
+    :param account_name: The name of the account to insert.
+    :type account_name: str
+    :param account_amount: The amount associated with the account
+                           to insert.
+    :type account_amount: Decimal
+    :return: None
+    """
+    async with AsyncSessionWithEnter(database.engine) as session:
+        account = Account(name=account_name, balance=account_amount)
+        session.add(account)
+        await session.commit()
+
+    return
+
+
+async def get_amount(account_id: int) -> Decimal:
+    """
+    Retrieve the amount associated with the given account ID from
+    the 'accounts' table.
+
+    :param account_id: The ID of the account to retrieve the amount for.
+    :type account_id: int
+    :return: The balance associated with the account.
+    """
+    async with AsyncSessionWithEnter(database.engine) as session:
+        account_obj = await session.execute(select(Account).where(
```

Comment:
Please, select retrieve balance directly within one query. There is no need to select full info of an account. Currently your SQL should look like this
```sql
SELECT * 
FROM Accounts
WHERE id == account_id
```
But you need
```sql
SELECT balance
FROM Accounts
WHERE id == account_id
```

Reply:
I changed it.